definitions/procedures/qpid_queues_clear.rb

```diff
@@ -0,0 +1,14 @@
+class Procedures::QpidQueuesClear < ForemanMaintain::Procedure
+  metadata do
+    for_feature :qpid_queues
+    description 'clear qpid queues'
+  end
+
+  def run
+    with_spinner('clear qpid queues') do |spinner|
+      total_queues_cleared = feature(:qpid_queues).clear_all
+      spinner.update "[#{total_queues_cleared.length}] queues cleared.\
```

Comment:
the `[14] queues cleared` would look better without `[]`, so it would be `14 queues cleared`